tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -42,16 +42,28 @@ copp/test_copp.py::TestCOPP::test_trap_config_save_after_reboot:
 #######################################
 decap/test_decap.py::test_decap[ttl=pipe, dscp=pipe]:
   skip:
-    reason: "Not supported on broadcom after 201911 release and mellanox all releases"
+    reason: "Not supported on broadcom after 201911 release, mellanox all releases and cisco-8000 all releases"
     conditions:
-      - "(asic_type in ['broadcom'] and release not in ['201811', '201911']) or (asic_type in ['mellanox'])"
+      - "(asic_type in ['broadcom'] and release not in ['201811', '201911']) or (asic_type in ['mellanox']) or (asic_type in ['cisco-8000'])"
 
 decap/test_decap.py::test_decap[ttl=pipe, dscp=uniform]:
   skip:
     reason: "Not supported on backend and broadcom before 202012 release"
     conditions:
       - "(topo_name in ['t1-backend', 't0-backend']) or (asic_type in ['broadcom'] and release in ['201811', '201911'])"
 
+decap/test_decap.py::test_decap[ttl=uniform, dscp=pipe]:
+  skip:
+    reason: "Not supported uniform ttl mode"
+    conditions:
+      -
+
+decap/test_decap.py::test_decap[ttl=uniform, dscp=uniform]:
+  skip:
+    reason: "Not supported uniform ttl mode"
+    conditions:
+      -
```

Comment:
Is this going to work? 

What if we don't supply `conditions` key here? If we don't supply any condition, it sounds like unconditionally add the `skip` mark. Is the conditional mark plugin support this syntax?

Reply:
Yes, I try this, if we don't supply any condition, it will unconditionally add the `skip` mark.

Reply:
Can you remove the `conditions` key here then?

Reply:
done